BPStatusBarAlert/Classes/BPStatusBarAlert.swift

```diff
@@ -20,6 +21,7 @@ public class BPStatusBarAlert: UIView {
         
     fileprivate var containerWindow: UIWindow?
     
+    fileprivate var deviceStatusBarHeight:CGFloat!
```

Comment:
How about not using forced unwrapping, what do you think of using **computed property**
ex)
```Swift
fileprivate var deviceStatusBarHeight: CGFloat {
    if UIDevice().userInterfaceIdiom == .phone && UIScreen.main.nativeBounds.height == 2436 {
        return 15
    }
    return 0
    }
```